packages/ui/src/services/global-zone/global-zone.service.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { IDisposable } from '@wendellhu/redi';
+import { createIdentifier } from '@wendellhu/redi';
+import type { Subject } from 'rxjs';
+
+export const IGlobalZoneService = createIdentifier<IGlobalZoneService>('univer.print-service');
```

Comment:
The token of the identifier remains unchanged.

```suggestion
export const IGlobalZoneService = createIdentifier<IGlobalZoneService>('univer.ui.global-zone');
```

And what does the service do? Could you please add some explanations in the code?

Reply:
Please note that that public services exposed to our users should be documented.